azure-ts-appservice-springboot/infrastructure/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
+/node_modules/
```

Comment:
Nit: no need in this file?

Reply:
It's a good idea to leave this since if someone copies just this project out of the examples repo. Also, I believe this was created by bootstrapping the project with `pulumi new ...`.